lib/search-bar/index.js

```diff
@@ -68,7 +70,19 @@ export default class SearchBarContainer extends Component {
       this.setState({ option: {} }); // clear selected option
     }
   }
-
+  placeholderSwitch () {
+    const placeholders = ['All inclusive...', 'Croatia...', 'Pool...', 'Thailand...', 'Vietnam...', 'Wifi...'];
+    this.setState({
+      placeholder: placeholders[counter]
+    });
+    counter++;
+    if (counter === placeholders.length) {
+      counter = 0;
+    }
+  }
+  componentDidMount () {
+    setInterval(this.placeholderSwitch.bind(this), 2000);
```

Comment:
will this over-write text input by the _person_ while they are typing? 
(_I don't think it will... but how do we check?_)